docs/organization.md

```diff
@@ -102,7 +102,7 @@ With all possible options:
       address: '0x1245', // address of the scheme to use; default is settings.ContributionReward
       params: {
         boolVote: organization.votingMachine, // votingMachine used to accept or reject contributions, default is organizaiton.votingMAchine
-        votePrec: 50, // percentage conditions under which a contribution is accepted, default is 50
+        votePerc: 50, // percentage conditions under which a contribution is accepted, default is 50
         orgNativeTokenFee: 0, // fee that is to be paid for proposing a contribution
         schemeTokenFee: 0, // fee that is to be paid for proposing a contribution
```

Comment:
schemeTokenFee is not needed any more and its not supported by the apis.

Reply:
Please save your comments on the documentation until you review the PR dedicated to documentation

Reply:
If organization.md include false information it is useless and just confused.
If you feel it should be included on other pr so remove it from this pr.

Reply:
@orenyodfat I am removing the entire "docs" folder in the forthcoming documentation PR.  I'll go ahead and do that in this one.

Reply:
Done